test/rest-api/binary.test.ts

```diff
@@ -0,0 +1,29 @@
+import * as path from 'path'
+import { TestAPI, runBrowserWith } from '../support/runBrowserWith'
+
+describe('REST: Binary response types', () => {
+  let test: TestAPI
+
+  beforeAll(async () => {
+    test = await runBrowserWith(path.resolve(__dirname, 'binary.mocks.ts'))
+  })
+
+  afterAll(() => {
+    return test.cleanup()
+  })
+
+  describe('given a binary response', () => {
+    it('should return a binary response', async () => {
+      const res = await test.request({
+        url: 'https://test.msw.io/binary',
```

Comment:
Could you please use `test.mswjs.io` (`msw` -> `mswjs`)? That way it points to the domain we own, not a third-party domain. 

> This is to ensure that tests have as less external dependencies as possible. Any link is potentially a dependency, as if it gets updated, it may affect the response and, thus, the test.

Reply:
I updated the domain. I got it from copying it from another test. Should I create an issue for updating all the test domains to use `test.mswjs.io`?

Reply:
Sorry, some tests were written at a different time, and may give a false example. 
No need, let's cover that as a separate change. Are you okay with that?

Reply:
Yes, absolutely.